doc.go

```diff
@@ -4,4 +4,4 @@
 // (https://substrait.io).
 package substraitgo
 
-//go:generate buf generate https://github.com/substrait-io/substrait.git#branch=v0.1.2
+//go:generate buf generate https://github.com/substrait-io/substrait.git#branch=master
```

Comment:
Should be main, not master.

Reply:
Bah, I had it right the first time and then confused myself lol

Fixed it now. It correctly says main